nox/virtualenv.py

```diff
@@ -39,6 +39,21 @@
 _SYSTEM = platform.system()
 
 
+def find_uv() -> str | None:
+    if uv := shutil.which("uv"):
```

Comment:
Can we drop Python 3.7? 😅

Reply:
I don't see why not IMO? It's well past EOL